ui/threejs/orbitDisplay.js

```diff
@@ -92,12 +92,17 @@ export const orbitDisplay = async (
     render();
   }).observe(page);
 
-  const updateGeometry = async (geometry) => {
+  const updateGeometry = async (
+    geometry,
+    { withGrid = true, fit = true} = {}
+  ) => {
     // Delete any previous dataset in the window.
     for (const { mesh } of datasets) {
       scene.remove(mesh);
     }
 
+    view.fit = fit;
```

Comment:
Rather than modifying the incoming structure, how about using { ...view, fit } instead of view, below?